indico/htdocs/js/indico/modules/sessions/types_dialog.js

```diff
@@ -0,0 +1,61 @@
+(function(global) {
```

Comment:
This file differs from `contributions/types_dialog.js` only in one class name. Shouldn't it be one file to make it more generic and not repeat the code?